target/classes/Pom_FeatureFiles/ResgisterAndLogin.feature

```diff
@@ -0,0 +1,12 @@
+Feature: User can register and Login
+Scenario: To verify whether user can register and login
+Given Access demoqa website Click on login
+When Click on New User button and enter all details
+And Click on Register button
+Then User should be registered sucessfully
+
+
+Scenario: To verify whether user can login
+Given Access demoqa website and click on login button
+When Enter username and password 
+Then User should be able to access the website   
```

Comment:
Hi Prabhat,

Please verify the title of the page also upon login.

Thanks,
Prabhat.

Reply:
Verify the title upon login step added  in feature file as per comments of Prabhat.